### DISCUSSION TITLE
Jamulus-3.8.0rc2 :: Reverb slider should be longer.

### DISCUSSION THREAD

**Discussion by ghost** [jamulussoftware/jamulus#1793] [UI Related]
**Describe the bug**

Reverb slider should be longer. Must have been overlooked, because I thought someone mentioned this.
**To Reproduce**

na
**Expected behavior**

na
**Screenshots**

na
**Operating system**

All, I suppose. Personally compiled Linux.
**Version of Jamulus**

Jamulus, Version 3.8.0rc2
**Additional context**

**Comment by dcorson-ticino-com** [comment]
The minimum size has been increased and the size increases when the it is less than 1/3rd of the free space.

**Comment by ghost** [comment]
> The minimum size has been increased and the size increases when the it is less than 1/3rd of the free space.

You are Exacty right. (hard to notice without knowing it is actually working)

Please change the properties to:
The minimum size has been increased and the size increases when it is less than 3/3rd of the free space.
(1/3 does not look good, whereas 100% does)

**Comment by dcorson-ticino-com** [comment]
IMHO just as overly big text is harder to read, overly big controls are also harder to see and use.

**Comment by hoffie** [comment]
> Reverb slider should be longer.

Do you mainly care about the visual layout or about the actual control (e.g. do you suggest we'd need more precise steps)?

**Comment by ghost** [comment]
> > Reverb slider should be longer.
> 
> Do you mainly care about the visual layout or about the actual control (e.g. do you suggest we'd need more precise steps)?

The visual layout. If a short slider is desired, maybe it should be in bottom third rather than the middle third.
You do know there are pleasing proportionalities for aesthetic design.

**Comment by gilgongo** [comment]
> If a short slider is desired, maybe it should be in bottom third rather than the middle third.

Sounds like it needs some discussion before this issue can be described for action.
